src/mlpack/core/util/sfinae_utility.hpp

```diff
@@ -182,6 +182,34 @@ struct NAME                                                                    \
       WithGreaterOrEqualNumberOfAdditionalArgs<MinN>::value;                   \
 };
 
+/**
+ * Constructs a template structure, which will define a boolean static variable
```

Comment:
Super pedantic style issue; this looks a little bit odd, do you think we could go with:

```c++
/**
 * Constructs a template structure, which will define a boolean static
 * variable, to true if the passed template parameter, has a member function
 * with the specified name. The check does not care about the signature or the
 * function parameters
```

Reply:
Sure. I assume you meant that each subsequent line should be longer or equal.